website/docs/quickstarts/duckdb-core-qs.md

```diff
@@ -0,0 +1,144 @@
+venv\Scripts\Activate.ps1
+python -m pip install --upgrade pip
+python -m pip install -r requirements.txt
+venv\Scripts\Activate.ps1
+dbt build
+dbt docs generate
+dbt docs serve
+```
+
+</TabItem>
+<TabItem value="codespace" label="GitHub Codespaces">
+
+#### Steps:
+
+1. Ensure you have [Codespaces](https://github.com/features/codespaces) enabled for your GitHub organization or turned on as a beta feature if you're an individual user.
+2. Open the [jaffle_shop_duckdb](https://github.com/dbt-labs/jaffle_shop_duckdb) repo in your browser.
+3. Click the green **Code** button near the top right of the repo's homepage (you may already be on it).
+4. Rather than cloning the repo like you normally would, click the **Codespaces** tab of the pop-out and then click **Create codespace on `duckdb`**.
+<div style={{maxWidth: '400px'}}>
+    <Lightbox src="/img/codespace-quickstart/open_in_codespaces.png" title="Open in Codespaces" />
+    </div>
+
+5. Wait for Codespaces to boot (this may take a few minutes).
+6. Select whether you'd like to use the Web IDE or open the codespace in your local environment.
+7. When the Codespaces opens, a **Task** pane will show up and call `dbt build` to show you how it's done.
+8. (Optional) Install the recommended extensions (like **dbt Power User extension**).
+9. Open up a new terminal and type: `dbt build`
+
+</TabItem>
+</Tabs>
+
+## Query the data
+
+The Jaffle Shop project seeds the data set to DuckDB, which can then be manually queried from the duckcli tool. 
+
+Launch the DuckDB command-line interface (CLI):
+
+```shell
+
+duckcli jaffle_shop.duckdb
+
+```
+
+Run a query at the prompt and exit:
+
+```sql
+
+select * from customers where customer_id = 42;
+exit;
+
+```
+
+## Browse the data
+
+There are a few options available to browse the data within DuckDB: 
+
+- [duckcli](https://pypi.org/project/duckcli/)
+- [DuckDB CLI](https://duckdb.org/docs/installation/?environment=cli)
+- [DBeaver SQL IDE for DuckDB](https://duckdb.org/docs/guides/sql_editors/dbeaver)
+
+## Next steps
+
+Now that you've got dbt Core, DuckDB, and the Jaffle Shop data up and running, you can explore dbt's capabilities. Refer to these materials to get a better understanding of dbt projects and commands:
+
+* The [About projects](/docs/build/projects) page guides you through the structure of a dbt project and its components.
+* The [dbt command reference](/reference/dbt-commands) explains the various commands available and what they do.
+* The [dbt Labs courses](https://courses.getdbt.com/collections) offer a variety of beginner, intermediate, and advanced learning modules designed to help you become a dbt expert. 
+* Once you see the potential of dbt and what it can do for your organization, sign up for a free trial of [dbt Cloud](https://www.getdbt.com/signup). It's the fastest and easiest way to deploy dbt today!
+* Check out the other [quickstart guides](/quickstarts) to begin integrating into your existing data warehouse.
```

Comment:
not sure where to put this so adding it here (apologies for being "out of place"!)

you should also add a new tile for this new qs. it's in the `website/vercel.json` file.